static/compiler-service.js

```diff
@@ -170,6 +175,7 @@ function handleRequestError(request, reject, xhr, textStatus, errorThrown) {
 
 CompilerService.prototype.submit = function (request) {
     var jsonRequest = JSON.stringify(request);
+    request.allowStoreCodeDebug = this.allowStoreCodeDebug;
```

Comment:
This needs to come before the json.stringify doesn't it?

Reply:
It sure does :) Good catch